vllm/executor/openvino_executor.py

```diff
@@ -286,39 +305,62 @@ def model_profile_run():
                     seq_len = (max_num_batched_tokens // max_num_seqs +
                             (group_id < max_num_batched_tokens % max_num_seqs))
                     seq_data = SequenceData([0] * seq_len)
-                    seq_num_blocks = (seq_len + self.cache_config.block_size - 1) // self.cache_config.block_size
-                    block_tables = torch.ones(seq_num_blocks, dtype=torch.int32)
                     seq = SequenceGroupMetadata(
                         request_id=str(group_id),
                         is_prompt=True,
                         seq_data={group_id: seq_data},
                         sampling_params=sampling_params,
-                        block_tables=block_tables,
+                        block_tables=None,
                         lora_request=None,
                     )
                     seqs.append(seq)
 
                 # Run the model with the dummy inputs.
+                self.model_runner.block_size = block_size
```

Comment:
how is it used?

Reply:
Hmm, I saw some complaints if block_size is not initialized, but I was not able to reproduce it now, so removed, thanks